apps/omg_watcher/priv/repo/migrations/20190806111817_alter_txoutputs_ethevents_make_many_to_many_relation.exs

```diff
@@ -0,0 +1,89 @@
+
+defmodule OMG.Watcher.DB.Repo.Migrations.AlterTxOutputsTableAddRootchainTxnHashDepositAndExitColumns do
+  use Ecto.Migration
+
+  # non-backward compatible migration, thus cannot use `change/0`
+
+  def up do
+    drop constraint(:txoutputs, "txoutputs_creating_deposit_fkey")
+    drop constraint(:txoutputs, "txoutputs_spending_exit_fkey")
+    drop constraint(:ethevents, "ethevents_pkey")
+
+    flush()
+
+    alter table(:ethevents) do
+      remove(:hash)
+      remove(:blknum)
+      remove(:txindex)
+
+      add(:rootchain_txhash, :binary, primary_key: true)
+
+      timestamps([type: :utc_datetime])
+    end
+
+    # how to do this in ecto correctly? do it manually
```

Comment:
I can see you've added corresponding `timestamps` to the schema - so I'd say it should just work...
Maybe it doesn't because we're not so `changeset` friendly... no idea. But `postgres` should handle this deficit ;)